.github/workflows/e2e-tests-workflow.yml

```diff
@@ -10,7 +12,7 @@ env:
   OPENSEARCH_DASHBOARDS_VERSION: '1.x'
   OPENSEARCH_VERSION: '1.3.0-SNAPSHOT'
 jobs:
-  test-with-security:
+  test-without-security:
```

Comment:
Don't see `test-with-security`, so we don't support e2e with security enabled? How about add some TODO ?

Reply:
Right, it's a known issue tracked in #117. Ideally the security snapshot artifacts get published to maven (similar to common-utils) and they can be pulled in and built in a new workflow.

Reply:
The previous solution was using snapshot docker images but those are only built for previous versions rather than upcoming versions. There is future infra plans to have these available for the upcoming versions which will eventually resolve all of these build-related issues.